public/js/reducers/management.js

```diff
@@ -26,6 +27,25 @@ export default function management(state, action) {
           debugMessage: action.error.debugMessage,
         },
       });
+    case actionTypes.GOT_SUBS_BY_PAY_METHOD:
+      return Object.assign({}, initialMgmtState, state, {
+        // This is short-lived data only relevant to the current
+        // operation.
+        viewData: {
+          affectedSubscriptions: action.subscriptions,
+          payMethodUri: action.payMethodUri,
+        },
+      });
+    case actionTypes.SHOW_CONFIRM_DEL_PAY_METHOD:
+      return Object.assign({}, initialMgmtState, state, {
+        // Ephemeral data for this operation only.
+        tab: 'pay-methods',
+        view: action.type,
+        viewData: {
+          operation: actionTypes.SHOW_CONFIRM_DEL_PAY_METHOD,
```

Comment:
Yeah, I'd missed one having ripped that out of the other places it was used already.